tracing/http_test.go

```diff
@@ -94,3 +95,77 @@ func TestHTTPSpanMiddleware(t *testing.T) {
 		t.Errorf("spans[0].TraceID = %s, want %s", got, traceID)
 	}
 }
+
+func TestHTTPSpanMiddlewareIgnoringPaths(t *testing.T) {
+	cfg := config.Config{
+		Backend: config.Zipkin,
+		Debug:   true,
+	}
+
+	// Create tracer with reporter recorder
+	reporter, co := FakeZipkinExporter()
+	defer reporter.Close()
+	oct := NewOpenCensusTracer(co)
+	defer oct.Finish()
+
+	if err := oct.ApplyConfig(&cfg); err != nil {
+		t.Errorf("Failed to apply tracer config: %v", err)
+	}
+
+	paths := []string{"/readyz"}
+	middleware := HTTPSpanMiddlewareIgnoringPaths(paths...)(&testHandler{})
+
+	testCases := map[string]struct {
+		path   string
+		traced bool
+	}{
+		"traced": {
+			path:   "/",
+			traced: true,
+		},
+		"ignored": {
+			path:   paths[0],
+			traced: false,
+		},
+	}
+	for n, tc := range testCases {
+		t.Run(n, func(t *testing.T) {
+			var lastWrite []byte
+			fw := fakeWriter{lastWrite: &lastWrite}
+
+			url := url.URL{
```

Comment:
Ugh -- not a fan of overloading the `url` name like this.

It looks like it's only used on the following line (141), so maybe just put it inline?

Reply:
I'd suggest just using `fmt.Sprintf("http://test.example.com/%s", tc.path)` (given the actual `http.NewRequest` interface), but I don't feel strongly.